js/ssai_pulse.js

```diff
@@ -132,6 +132,9 @@ OO.Ads.manager(function(_, $)
     {
       amc = adManagerController;
 
+      // Request embed code provider metadata
+      amc.willRequireEmbedCodeMetadata();
```

Comment:
@orobles-ooyala can you create a unit test that checks that willRequireEmbedCodeMetadata is fired?